pkgs/by-name/fi/finamp/package.nix

```
@@ -0,0 +1,73 @@
+{ lib
+, flutter
+, mpv
```

```suggestion
, mpv-unwrapped
```

Not so sure, but usually we don't use wrapped ones if we need the library

---

Fixed